bin/redshift/delete_aws_redshift.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+if [[ ! -f $DIR/.cluster-identifier ]];  then
```

Comment:
always put variables in `"` quotes. the code here would break if the path contained a whitespace

also, put variable name in `${...}` braces (i am following https://google.github.io/styleguide/shellguide.html#variable-expansion here)